### ISSUE TITLE
Make labels clearer for CPU average and max metrics

### ISSUE THREAD

**Issue by briansmith0** [cockpit-project/cockpit#18999]
Page: Overview / "View metrics and history"

Currently, the metrics page shows two CPU graph bars: one for the average of all CPU's, and one for the CPU core with the most usage.  There is text above these 2 CPU graphs that shows  "<#> CPUs  average: _percent_ max _percent_".  

However, it isn't very clear that these labels line up with the two CPU graph bars currently.  In addition, there is no indication of which CPU core is being shown as the top core.  It would be helpful to be able to see if the top core shown is changing while watching the graphs.  

![Screenshot from 2023-06-22 14-53-01 (copy)](https://github.com/cockpit-project/cockpit/assets/60192516/fdd29eff-3d73-4bf8-9842-cc76e1b0aab5)

**Comment by martinpitt** [comment]
@garrett can you please have a look at this? We've been iterating over this several times. We can't/shouldn't put much text there, as it would lead to overflowing in narrower scenarios, like

![image](https://github.com/cockpit-project/cockpit/assets/200109/71ee89db-18bc-4229-b253-46131009a557)

Perhaps "Most utilized core" could become "top:", and the more verbose strings go into the popover?

**Comment by garrett** [comment]
Does knowing the core actually help at all? What if there are multiple cores around the same value? What happens when the max core changes every refresh?

We do have "view all CPUs" which shows all the CPUs if you really want to know which cores are have the highest load.

> However, it isn't very clear that these labels line up with the two CPU graph bars currently.

The two bars have a pixel width apart (right next to each other) and are labelled with a heading directly above.

It's super common in all UIs, even in Cockpit, to have similar items grouped together based on space.

It's part of the "Gestalt principles" of design (which are quite fundamental), where objects together are more than the sum of their parts.

https://www.smashingmagazine.com/2014/03/design-principles-visual-perception-and-the-principles-of-gestalt/

**Comment by briansmith0** [comment]
This issue was created based on feedback I received from a RHEL user, and I agreed that it is not clear based on a first glance at the page.  I just had someone else (who's never used Cockpit, but is familiar topics like CPU usage) take a look at that page.  At first glance, they weren't sure what the 2 graphs represented.  They figured it out by looking at the % value of the labels above and figuring out that the graphs below were at those same percents, so the labels above likely lined up with the graphs in a corresponding order -- but this took some thought process and time to figure out.

In summary - with the current page it seems like people can figure it out, but it isn't intuitively clear without some thought process/time.

**Comment by briansmith0** [comment]
One thing I noticed as well is if you have lower CPU usage, the graphs and the labels can be far apart, which might contribute to this not being intuitive for some people:  

![Screenshot from 2023-06-27 11-54-40](https://github.com/cockpit-project/cockpit/assets/60192516/49d612da-7493-47e5-abc2-f1786b5d2d86)

**Comment by garrett** [comment]
We could probably stack the bars by making the average one (which would always be less than max) have a transparent gutter and have both be at the same place on the page. It would look like this:

![image](https://github.com/cockpit-project/cockpit/assets/10246/567f02e3-54ff-40df-b487-84c885a9b5c1)

![image](https://github.com/cockpit-project/cockpit/assets/10246/082c97e7-8a87-4681-92cc-66ad5d01a34a)

The stacked bars would _usually_ look like this:

![image](https://github.com/cockpit-project/cockpit/assets/10246/e951393c-5f61-4864-a219-c06622965a6e)
